elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ProviderAttributeDefinitions.java

```diff
@@ -218,10 +236,39 @@ class ProviderAttributeDefinitions {
                     .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
                     .build();
 
+    protected static final SimpleAttributeDefinition AUTHENTICATION_REQUEST_FORMAT =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.AUTHENTICATION_REQUEST_FORMAT, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setDefaultValue(new ModelNode("oauth2"))
+                    .setAllowedValues("oauth2", "request", "request_uri")
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .build();
+
+    protected static final SimpleAttributeDefinition REQUEST_OBJECT_ENCRYPTION_ALGORITHM =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.REQUEST_OBJECT_ENCRYPTION_ALGORITHM, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .build();
+
+    protected static final SimpleAttributeDefinition REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM =
+            new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM, ModelType.STRING, true)
+                    .setAllowExpression(true)
+                    .setValidator(new StringLengthValidator(1, Integer.MAX_VALUE, true, true))
+                    .build();
```

Comment:
@PrarthonaPaul Just a minor, if the REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM  attribute requires configuration of REQUEST_OBJECT_ENCRYPTION_ALGORITHM attribute as well, you might be able to use this here also:

`.setRequires(ElytronOidcDescriptionConstants.REQUEST_OBJECT_ENCRYPTION_ALGORITHM)`

Reply:
Fixed! 
Added it to both REQUEST_OBJECT_ENCRYPTION_ALGORITHM and REQUEST_OBJECT_CONTENT_ENCRYPTION_ALGORITHM